FlagsmithClient/Classes/Flagsmith.swift

```diff
@@ -268,6 +268,21 @@ public final class Flagsmith: @unchecked Sendable {
             completion(result.map(\.traits))
         }
     }
+    
+    /// Get flags for an identity and set traits in the same call
+    ///
+    /// - Parameters:
+    ///   - traits: Traits to be created or updated
+    ///   - identity: ID of the user
+    ///   - completion: Closure with Result which contains a list of Flags in case of success or Error in case of failure
+    public func getIdentityFlags(_ traits: [Trait],
+                          forIdentity identity: String,
+                          completion: @Sendable @escaping (Result<[Flag], any Error>) -> Void)
```

Comment:
You'll have to fix this swift lint things. If you run it locally there's --fix option that sorts most of it

Reply:
Thanks will sort